.gitignore

```diff
@@ -13,4 +13,7 @@
 /captures
 .externalNativeBuild
 .cxx
-local.properties
\ No newline at end of file
+local.properties
+awsconfiguration.json
+amplifyconfiguration.json
+/buildSrc/build
```

Comment:
Can we remove jsons from here, if they are present?

Reply:
done